src/main/java/dan200/computercraft/shared/pocket/apis/PocketAPI.java

```diff
@@ -150,6 +162,78 @@ public void shutdown()
                         return null;
                     }
                 } );
+            case 2:
+                // beep
+                float volume = 1f;
+                float pitch = 1f;
+
+                // Check if arguments are correct
+                
+                if ( arguments.length > 0 )
+                {
+                    if ( arguments[0] != null && !(arguments[0] instanceof Double) )  // Arg 0 wrong type
+                    {
+                        throw new LuaException("Expected number (optional), number (optional)" );
+                    }
+
+                    volume = arguments[0] != null ? ((Double) arguments[0]).floatValue() : 1f;
+
+                }
+
+                if ( arguments.length > 1 )
+                {
+                    if ( arguments[1] != null && !(arguments[1] instanceof Double) )  // Arg 1 wrong type
+                    {
+                        throw new LuaException("Expected string, number (optional), number (optional)" );
+                    }
+                    pitch = arguments[1] != null ? ((Double) arguments[1]).floatValue() : 1f;
+                }
+                
+                if( !(m_computer.getEntity() instanceof EntityPlayer) )
+                {
+                    throw new LuaException( "Cannot find player" );
+                }
+				
+                ResourceLocation resourceName = new ResourceLocation( "block.note.bell" );
+                
+                //Fallback to harp if this version of minecraft don't have bell.
+                if ( !( SoundEvent.REGISTRY.containsKey( resourceName ) ) )
+                {
+                    resourceName = new ResourceLocation( "block.note.harp" );
+                }
+                if ( m_clock - m_lastBeepTime >= 1 ) //Once on tic only
+                {
+                    if ( SoundEvent.REGISTRY.containsKey(resourceName) )
+                    {
```

Comment:
Legacy code from #237 -- you can safely remove this `if` as you are referencing a known to exist `ResourceLocation`